manifests/application/datamgmt.pp

```diff
@@ -38,17 +31,12 @@
     if $package_name == 'lightblue-data-mgmt-saml-auth' {
         include lightblue::authentication::saml
 
+        if $app_uri == undef {
```

Comment:
This won't fail unless someone sets it explicitly.  Is saml still a thing for the data management app?